transformer_sidecar/src/transformer_sidecar/transformer.py

```diff
@@ -120,7 +120,27 @@ def fill_stats_parser(stats_parser_name: str, logfile_path: Path) -> Transformer
     return globals()[stats_parser_name](logfile_path)
 
 
+def prepend_xcache(file_paths):
+    # if CACHE_PREFIX is given, returns file paths prefixed with
+    # a cache determined by the filename. Cache list is comma separated.
+    prefix = os.environ.get('CACHE_PREFIX', '')
+    if not prefix:
+        return file_paths
+    # split the string into a list of xcaches. strip in case someone adds spaces
```

Comment:
This is describing the code, but I still have no idea how we get from the hash/hex digest to the Xchache. Maybe a through description in the ServiceX `values.yaml` file for the `cachePrefix` setting. It seems like there is some external configuration that this is referring to but it is not apparent to me

Reply:
I am not sure I understand your question... we now expect cachePrefix setting to be eg. "cache1.uc.edu" or "cache1.uc.edu,cache2.uc.edu,whatever.edu" this string will be split into a list of prefixes. only one of them will be prepended to the file path. Exactly which one depends on the hash of the filename.
Does this explain?